src/client/components/Navbar.css

```diff
@@ -11,6 +11,19 @@
     img {
       height: 70px;
     }
+
+    .logo-black {
+      display: none;
+    }
+    
```

Comment:
maybe also add 
```
.logo {
  display: block;
}
```

so that it's clear that the `@media` query directly flip-flops the behavior